e2etest/zt_basic_copy_sync_remove_test.go

```diff
@@ -77,6 +77,63 @@ func TestBasic_CopyUploadLargeBlob(t *testing.T) {
 	}, EAccountType.Standard(), EAccountType.Standard(), "")
 }
 
+func TestBasic_CopyUploadLargeAppendBlob(t *testing.T) {
+	dst := common.EBlobType.AppendBlob()
+	src := dst
+
+	RunScenarios(t, eOperation.Copy(), eTestFromTo.Other(common.EFromTo.BlobBlob(), common.EFromTo.LocalBlob()), eValidate.Auto(), anonymousAuthOnly, anonymousAuthOnly, params{
+		recursive: true,
+		blobType:  src.String(),
+	}, &hooks{
+		beforeRunJob: func(h hookHelper) {
+			h.CreateFile(f("test.txt", with{blobType: dst}), false)
+		},
+		afterValidation: func(h hookHelper) {
+			props := h.GetDestination().getAllProperties(h.GetAsserter())
+			bprops, ok := props["test.txt"]
+			h.GetAsserter().Assert(ok, equals(), true)
+			if ok {
+				h.GetAsserter().Assert(bprops.blobType, equals(), dst)
+			}
+		},
+	}, testFiles{
+		defaultSize: "100M",
+
+		shouldTransfer: []interface{}{
+			f("test.txt", with{blobType: src}),
+		},
+	}, EAccountType.Standard(), EAccountType.Standard(), src.String()+"-"+dst.String())
+}
+
+func TestBasic_CopyUploadLargeAppendBlobBlockSizeFlag(t *testing.T) {
+	dst := common.EBlobType.AppendBlob()
+	src := dst
+
+	RunScenarios(t, eOperation.Copy(), eTestFromTo.Other(common.EFromTo.BlobBlob(), common.EFromTo.LocalBlob()), eValidate.Auto(), anonymousAuthOnly, anonymousAuthOnly, params{
+		recursive:   true,
+		blobType:    src.String(),
+		blockSizeMB: 100, // 100 MB
+	}, &hooks{
+		beforeRunJob: func(h hookHelper) {
+			h.CreateFile(f("test.txt", with{blobType: dst}), false)
+		},
+		afterValidation: func(h hookHelper) {
+			props := h.GetDestination().getAllProperties(h.GetAsserter())
+			bprops, ok := props["test.txt"]
+			h.GetAsserter().Assert(ok, equals(), true)
+			if ok {
+				h.GetAsserter().Assert(bprops.blobType, equals(), dst)
+			}
+		},
+	}, testFiles{
+		defaultSize: "100M",
```

Comment:
should we make the file a little larger than a single block?

Reply:
Not sure if I understood your question correctly, but 100 MB is the max size so anything larger will fail and error out.

Reply:
that parameter is the size of the file, is 100MB the max file size for append blob?

Reply:
Yes, it is.

Reply:
It looks like the max append blob size was several 100GB and now it should be larger due to the increase in block size. https://learn.microsoft.com/en-us/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs#about-append-blobs

Reply:
Updated the file size to 101 MB! I didn't make it too big to avoid lagging in tests.